roles/openshift_health_checker/openshift_checks/mixins.py

```diff
@@ -0,0 +1,19 @@
+from openshift_checks import OpenShiftCheckException
+
+
+class NotContainerized(object):
```

Comment:
What I have identified so far is that we need to differentiate between `failed`, `passed` or `skipped` (N/A). `is_active` answers whether to skip the check.

What problem is the alternative trying to solve / improve?

I see it increases the API surface and spreads the knowledge about `task_vars` to even more methods.

From a code maintenance point of view, it adds more burden to check authors. Each check would need to overwrite one or more `supports_*` method, compared to adding the right set of mixins.

A new `supports_system_containers` method requires changing multiple places, failing the Open-Closed Principle...